### PR TITLE
Convert to .NET Core 3.1

### PR DESCRIPTION
**Pull request by cartermp**
3.0 is unsupported

### CODE REVIEW

**Comment by jmarolf** [comment]
Shouldn't we just pick the one based on what is installed with the SDK? otherwise we need to update this list forever

File: src/MSBuild.Conversion.Facts/MSBuildFacts.cs [hunk 1]
```diff
@@ -191,7 +191,7 @@ public static class MSBuildFacts
         public const string DefaultSDKAttribute = "Microsoft.NET.Sdk";
         public const string LowestFrameworkVersionWithSystemValueTuple = "net47";
         public const string SharedProjectsImportLabel = "Shared";
-        public const string NetCoreAppTFM = "netcoreapp3.0";
+        public const string NetCoreAppTFM = "netcoreapp3.1";
```
Comment (CyrusNajmabadi) [inline src/MSBuild.Conversion.Facts/MSBuildFacts.cs]:
```suggestion
        public const string NetCoreAppTFM = "netcoreapp3.2";
```

**Comment by cartermp** [comment]
yeah but we can just do that later when we're updating this for <the thing>

File: src/MSBuild.Conversion.Facts/MSBuildFacts.cs (hunk 1 above)
Comment (CyrusNajmabadi) [inline src/MSBuild.Conversion.Facts/MSBuildFacts.cs]:
```suggestion
        public const string NetCoreAppTFM = "netcoreapp3.3";
```

Comment (CyrusNajmabadi) [inline src/MSBuild.Conversion.Facts/MSBuildFacts.cs]:
```suggestion
        public const string NetCoreAppTFM = "netcoreapp3.14159";
```

**Comment by cartermp** [comment]
I refuse to use anything other than github's web ui for this

**Comment by cartermp** [comment]
yeet

**Pull request merged**